src/util/metadata.js

```diff
@@ -603,6 +603,10 @@ export default {
     keywords: 'help life ring support',
     categories: ['ui'],
   },
+  Lifeascent: {
+    keywords: '',
+    categories: ['ui'],
+  },
```

Comment:
@dexterca may have thoughts on this as well, but I feel like this should be camel case instead.  If that's what we want, we'd need to rename the following in Chromicons:

```
src/raw/lined/lifeascent.svg
```

to

```
src/raw/lined/life-ascent.svg
```

Doing so will make this `LifeAscent` instead which I think reads a bit easier

Reply:
I like `life-ascent.svg`

Reply:
So, I'd do that before merging this PR? Or merge this and then edit the chromicons and update the version?

Reply:
Doesn't make a huge difference, but I'd probably update chromicons first, then come back and update this PR